README.md

```diff
@@ -230,6 +230,8 @@ Ruby:
 Rust:
 
 - rustc
+- cargo (call with `Neomake! cargo`)
+- [clippy](https://github.com/Manishearth/rust-clippy) (call with `Neomake! clippy`, needs nightly rust, supports [rustup](https://github.com/rust-lang-nursery/rustup.rs))
```

Comment:
> needs nightly rust

That will certainly change, so better have a more relative version/naming here..